scripts/visualize/visualize_utils.R

```diff
@@ -167,4 +167,11 @@ clip_sp.Spatial <- function(sp, xlim, ylim, ..., clip.fun = rgeos::gIntersection
 #' set up the basic plot par for a map
 set_sp_plot <- function(){
   par(mai=c(0,0,0,0), omi=c(0,0,0,0), xaxs = 'i', yaxs = 'i')
-}
\ No newline at end of file
+}
+
+#' script to turn the dataviz into a thumbnail
+#' 
+visualize.map_thumbnail <- function(viz){
+  data <- readDepends(viz)
+  file.copy(from = data[[1]], to = viz[['location']], overwrite = TRUE)
```

Comment:
I guess this is ok for now, but aren't we going to end up just publishing a png we made by hand?  There isn't an automated thumbnail generator yet is there?

Reply:
Yes, I'm working on that part now. I was just trying to get this up before viz.yaml changed too much and I'd have to deal with a bunch of merge conflicts.